.github/workflows/UpdateALGoOnOfficialBranches.yaml

```diff
@@ -0,0 +1,40 @@
+name: Update AL-Go On Official Branches
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "55 08 * * 3" # Run at 08:55 UTC on Wednesday
+
+jobs:
+  GetBranches:
+    name: Get Official Branches
+    runs-on: ubuntu-latest
+    outputs:
+      UpdateBranches: ${{ steps.OfficialBranches.outputs.OfficialBranches }}
+    steps:
+      - name: Get Official Branches
+        id: OfficialBranches
+        uses: microsoft/BCApps/.github/actions/GetOfficialBranches@main
+
+  UpdateALGo:
+    name: "[${{ matrix.branch }}] Update AL-Go"
+    runs-on: ubuntu-latest
+    environment: Official-Build
+    needs: GetBranches
+    strategy:
+      matrix:
+        branch: ${{ fromJson(needs.GetBranches.outputs.UpdateBranches) }}
+      fail-fast: false
+    steps:
+      - name: Checkout
```

Comment:
Why do we need to check out the repo?

Reply:
Now that the action is not local, probably not. Will check.

Reply:
Because some of the automations need to run on the code (update files, etc).

Reply:
But true, in UpdateALGoOnOfficialBranches.yaml the checkout could be omitted.

Reply:
Removed the "Checkout" step.